submissions/TaniaBondarenko/friends-app/app.js

```diff
@@ -0,0 +1,161 @@
+const NUM_OF_FRIENDS = 20;
+const FRIENDS = document.querySelector(".friends");
+let allFriends;
+let friendsForFilter;
+let isFiltered = false;
+let justMale;
+let justFemale;
+
+function fetchFriends() {
+  const api_url = `https://randomuser.me/api/?results=${NUM_OF_FRIENDS}`;
```

Comment:
Violation of naming convention. All your code on `camelCase` ;)

Reply:
My bad